src/Sentry/Internal/MainSentryEventProcessor.cs

```diff
@@ -56,6 +59,11 @@ public SentryEvent Process(SentryEvent @event)
                 @event.Contexts[CurrentUiCultureKey] = currentUiCultureMap;
             }
 
+#if NETCOREAPP3_0_OR_GREATER
+            @event.Contexts[IsDynamicCodeCompiledKey] = RuntimeFeature.IsDynamicCodeCompiled;
+            @event.Contexts[IsDynamicCodeSupportedKey] = RuntimeFeature.IsDynamicCodeSupported;
```

Comment:
![image](https://user-images.githubusercontent.com/8229322/147567925-f6167630-e26f-4c47-b46e-b04de6641ab0.png)
This will not work. And I understand that the context struct makes you believe it accepts an object as a parameter but in reality, it expects a dictionary 😅

I would suggest the following change to your code in order to attach the new data.

```suggestion
            @event.Contexts[IsDynamicCodeKey] = new Dictionary<string, bool>
            {
                { IsDynamicCodeCompiledKey, RuntimeFeature.IsDynamicCodeCompiled },
                { IsDynamicCodeSupportedKey, RuntimeFeature.IsDynamicCodeSupported }
            };
```

![image](https://user-images.githubusercontent.com/8229322/147568477-d988da3b-7ad2-428f-9fc2-5323e563fbcc.png)

Reply:
@lucas-zimerman thanks